README.rst

```diff
@@ -3,13 +3,12 @@ quick-csr
 
 A command-line tool to quickly generate *Certificate Signing Requests* (CSR_),
 targeted to DAUs and busy people. The major desktop operating systems are
-supported.
+supported. **Python 3.5 or later is required.**
```

Comment:
detailed system requirements are usually discussed in dedicated sections or in the installations sections of software documentations. such conventions make it easy to find what one is looking for. in an introductary or feature section such apsects are only communicated when it's an uber feature that only few mastered so far.

Reply:
Since Python is needed for using quick-csr as zip-application, only adressing system requirements under "installation" seems unconvenient.
Allthough it may cause confusion amongst some users, I suggest to point out the need for Python under "installation" AND "usage of quick-csr as zip application".

Reply:
Good point, solve as you like.